src/main/java/org/dataone/bookkeeper/jdbi/UsageStore.java

```diff
@@ -36,20 +43,85 @@
             "u.instanceId, " +
             "u.quantity, " +
             "u.status " +
-        "FROM usages u " +
-        "INNER JOIN quotas q ON u.quotaId = q.id ";
+        "FROM usages u ";
 
     /** The full query */
     String SELECT_ALL = SELECT_CLAUSE;
 
-    /** Select by identifier */
-    String SELECT_ONE = SELECT_CLAUSE + "WHERE u.id = :id";
+    /** Select by identifer */
+    String SELECT_BY_ID = SELECT_CLAUSE +
+            "WHERE u.id = :id";
+
+    /** Select by instance identifier */
+    String SELECT_ONE = SELECT_CLAUSE + "WHERE u.instanceId = :instanceId";
```

Comment:
By convention I was using `SELECT_ONE` to select by row id, so let's change this to:
```suggestion
    String SELECT_INSTANCE = SELECT_CLAUSE + "WHERE u.instanceId = :instanceId";
```